spaces/S000156/properties/P000050.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000156
+property: P000050
+value: true
+refs:
+- mathse: 4673615
+  name: What separation properites are satisfied by the Arens space?
+---
+
+Basic open sets in a convergent sequence are open; this passes to {S000156}.
```

Comment:
You meant "are closed" I suppose.

Also, can't we just use "{S156}" for simplicity?  Easier to read when we edit things.